src/index.ts

```diff
@@ -28,6 +28,11 @@ export interface PoolOptionsBase {
    * Number of connections to store in the pool
    */
   poolSize: number;
+  /**
+   * The minimum number of connections to store in the pool.
+   * These connections will be lazily opened.
```

Comment:
I'm thinking that if there's a minimum pool size, either the connections should be opened immediately or there should be a flag to control if the connections should be opened immediately or lazily.